From 238a14fe30624b62dce946c28c76fdad4e1388ff Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Martin=20Storsj=C3=B6?= <martin@martin.st>
Date: Fri, 5 May 2023 11:05:26 +0000
Subject: [PATCH 4/8] [test] [llvm-config] Assume unix style lib names on mingw
 targets

Assume the MSVC style naming only for "windows-msvc" targets.

Differential Revision: https://reviews.llvm.org/D149998
---
 llvm/test/tools/llvm-config/system-libs.test         | 2 +-
 llvm/test/tools/llvm-config/system-libs.windows.test | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/llvm/test/tools/llvm-config/system-libs.test b/llvm/test/tools/llvm-config/system-libs.test
index 708ce3ef99a4..ac747bf4e8c8 100644
--- a/llvm/test/tools/llvm-config/system-libs.test
+++ b/llvm/test/tools/llvm-config/system-libs.test
@@ -1,6 +1,6 @@
 RUN: llvm-config --link-static --system-libs Support 2>&1 | FileCheck %s
 REQUIRES: static-libs
-UNSUPPORTED: system-windows
+UNSUPPORTED: target={{.*-windows-msvc}}
 CHECK: -l
 CHECK-NOT: error
 CHECK-NOT: warning
diff --git a/llvm/test/tools/llvm-config/system-libs.windows.test b/llvm/test/tools/llvm-config/system-libs.windows.test
index 55d1fbea9788..471ed871c304 100644
--- a/llvm/test/tools/llvm-config/system-libs.windows.test
+++ b/llvm/test/tools/llvm-config/system-libs.windows.test
@@ -1,6 +1,6 @@
 RUN: llvm-config --link-static --system-libs Support 2>&1 | FileCheck %s
 REQUIRES: static-libs
-REQUIRES: system-windows
+REQUIRES: target={{.*-windows-msvc}}
 CHECK-NOT: -l
 CHECK: psapi.lib shell32.lib ole32.lib uuid.lib advapi32.lib
 CHECK-NOT: error
-- 
2.34.1

